numba_cuda/numba/cuda/tests/cudapy/test_debuginfo.py

```diff
@@ -298,6 +298,14 @@ def f(x, y):
         match = re.compile(pat).search(llvm_ir)
         self.assertIsNotNone(match, msg=llvm_ir)
 
+    def test_kernel_args_types(self):
+        self._test_kernel_args_types()
+
+    def test_kernel_args_types_dump(self):
+        # see issue#135
+        with override_config('DUMP_LLVM', 1):
+            self._test_kernel_args_types()
```

Comment:
This needs the output capturing to prevent the test suite dumping everything to stdout (which pollutes the CI logs and the terminal output when running locally:

```suggestion
            with captured_stdout():
                self._test_kernel_args_types()
```

`captured_stdout` can be imported from `numba.tests.support`.

Reply:
Thank you Graham for pointing this out. Tweaking another commit.